extension.json

```diff
@@ -77,6 +77,10 @@
                   "type": "string",
                   "enum": ["web", "device", "environment", "placeContext"]
                 }
+              },
+              "onBeforeEventSend": {
+                "type": "string",
+                "pattern": "^%[^%]+%$"                  
```

Comment:
Maybe a comment here would help why that pattern is necessary.

Reply:
He can't comment here unfortunately since it's a JSON file. :(